internal/pkg/tfconf/output.go

```diff
@@ -4,6 +4,7 @@ package tfconf
 type Output struct {
 	Name        string   `json:"name" yaml:"name"`
 	Description String   `json:"description" yaml:"description"`
+	Value       string   `json:"value,omitempty" yaml:"value,omitempty"`
```

Comment:
This will have some limitation as it will only render the value as `string` and also omit them if they are empty, which means the schema of json will be different based on the selected flags. You don't need to worry about this, we will have to circle back to it when the above comments are done.

Also I still need to understand how `terraform output --json` exactly works, I've briefly tested something and got different results based on the version of terraform binary I used.

Reply:
When you run the latest terraform, are you getting errors like I mentioned here?  #183 
 https://github.com/segmentio/terraform-docs/issues/183

Reply:
No it wasn't like #183 I was getting different `type` and `value`. Some of them were plain `string` and `"some value"`, some other were `["list", "string"]` and `["val1", "val2"]` and more. I haven't had enough time to try to understand a pattern.

Reply:
So the behavior seems to be as:

- terraform 0.12.x: `type` of output is always `[]interface{}`
- terraform 0.11.x and prior: `type` of outputs is always `string`

In our case it doesn't make any difference at the moment because we're not showing the _type_ of output and we only need to extract it properly, which by using `Type interface{}` in `TerraformOutput` struct this will be fine.